http.d.ts

```diff
@@ -34,29 +35,54 @@ export interface IHttpParam {
   };
 }
 
+declare const enum HttpParamStyles {
+  simple = 'simple',
+  matrix = 'matrix',
+  label = 'label',
+  form = 'form',
+  spaceDelimited = 'spaceDelimited',
+  pipeDelimited = 'pipeDelimited',
+  deepObject = 'deepObject',
```

Comment:
In general the TS world uses `PascalCase` for enum names - should we follow that?

Reply:
I am good with that!

Reply:
resolved